src/SFML/Window/Win32/WindowImplWin32.hpp

```diff
@@ -110,6 +110,22 @@ class WindowImplWin32 : public WindowImpl
     ////////////////////////////////////////////////////////////
     void setSize(const Vector2u& size) override;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the minimum size of the window, preventing it from being user-resized to smaller than this. Defaults to 0,0
```

Comment:
```suggestion
    /// \brief Sets a minimum window rendering region size
```
Let's use the same `\brief` as the other implementations.